flow-typed/box-annotations.js

```diff
@@ -106,45 +106,70 @@ type Annotation = {
     location?: Location,
     threadNumber?: string,
     comments: Comments,
-    createdBy?: User, 
+    createdBy?: User,
     createdAt?: string,
     canAnnotate: boolean,
-    canDelete: boolean
-}
+    canDelete: boolean,
+};
 
 type Options = {
     apiHost: string,
     fileId: string,
     token: string,
     anonymousUserName: string,
-    permissions: BoxItemPermissions
+    permissions: BoxItemPermissions,
 };
 
 type AnnotationDetails = {
     threadID: string,
     type: AnnotationType,
-    location: Location
+    location: Location,
 };
 
 type BoxUser = {
     type: 'user',
     id: string,
     login: string,
     name: string,
-    profile_image: string
-}
+    profile_image: string,
+};
 
 type AnnotationData = {
     id: string,
     details: AnnotationDetails,
     item: BoxFileVersion,
     message: string,
     permissions: AnnotationPermissions,
-    created_by: BoxUser, 
+    created_by: BoxUser,
     created_at: string,
     modified_at: string,
-    thread: string
+    thread: string,
+};
+
+type intlMessage = {
+    id: string,
+    description: string,
+    defaultMessage: string,
+};
+
+export type IntlOptions = {
+    intlLocaleData: Array<any>,
+    provider: Object,
+    language: string,
+}
+export type IntlContext = {
```

Comment:
We should probably start using Typescript if we're going to rely on types heavily.